### ISSUE TITLE
Error: length(kal$bootstrap) > 0 is not TRUE

### ISSUE THREAD

**Issue by tinyheero** [pachterlab/sleuth#21]
I used kallisto 0.42.3 to quantify transcript abundance. I followed the sleuth tutorial using version 0.27.2 and when I got to `so <- sleuth_fit(so)` step, I get the following error:

```
> so <- sleuth_fit(so)
summarizing bootstraps
Error: length(kal$bootstrap) > 0 is not TRUE
```

Is this related to incompatible versions of sleuth and kallisto?

**Comment by pmelsted** [comment]
Was kallisto run with bootstraps, i.e. `-b` parameter?

**Comment by tinyheero** [comment]
Didn't realize that -b defaults to 0 in kallisto. Running it with `-b 100` now. Thanks.

**Comment by pmelsted** [comment]
@pimentel we should probably warn about this with a better error message.

**Comment by pimentel** [comment]
doh. meant to do this in `check_kal_pack`. will do in next version. thanks for the report

**Comment by pimentel** [comment]
This has been addressed in pull request #28. It will go in the next version of sleuth.

Thanks again for the bug report.
